examples/kitty/board/odroidc4/kitty.system

```diff
@@ -75,9 +70,9 @@
     <memory_region name="i2c_micropython_data_region" size="0x200_000" page_size="0x200_000"/>
 
     <!-- shared memory for micropython/nfs queue -->
-    <memory_region name="shared_nfs_micropython" size="0x4000000" />
-    <memory_region name="nfs_command_queue" size="0x8_000" />
-    <memory_region name="nfs_completion_queue" size="0x8_000" />
+    <memory_region name="shared_nfs_micropython" size="0x200_000" />
```

Comment:
Why did the size of `shared_nfs_micropython` change?

Reply:
Actually all these MRs changed in size?

Reply:
These regions were increased 3 weeks ago in this commit: https://github.com/au-ts/lionsos/commit/914c9eb35373469a618df4221759aed925aee493. This must of changed during the rebase. I will change these to the updated sizes.